__tests__/endpoints.test.js

```diff
@@ -90,3 +119,47 @@ describe('GET /api/articles/:article_id', () => {
       });
   });
 });
+
+describe('PATCH /api/articles/:article_id', () => {
+  test('STATUS: 200, Returns updated object in new state', () => {
+    return request(app)
+      .patch('/api/articles/1')
+      .send({
+        inc_votes: 50,
+      })
+      .expect(200)
+      .then(({ body: { article } }) => {
+        expect(article).toEqual(
+          expect.objectContaining({
+            article_id: 1,
+            title: 'Living in the shadow of a great man',
+            topic: 'mitch',
+            author: 'butter_bridge',
+            body: 'I find this existence challenging',
+            created_at: '2020-07-09T20:11:00.000Z',
+            votes: 150,
+          })
+        );
+      });
+  });
+
+  test('STATUS: 404, Returns not found if ID is invalid', () => {
```

Comment:
I'd reword these sad path tests to be specific about what we're testing for as these two tests read as returning different messages if the id is invalid. Could maybe use wording like 'VALID but NON-EXISTENT article_id' and 'INVALID article_id'

Reply:
Have updated test names to reflect tests correctly